arangod/Aql/Executor/CalculationExecutor.cpp

```diff
@@ -115,6 +115,11 @@ CalculationExecutor<calculationType>::produceRows(
     // by exterior.
     TRI_ASSERT(!shouldExitContextBetweenBlocks() || !_hasEnteredExecutor ||
                state == ExecutorState::HASMORE);
+
+    _killCheckCounter = (_killCheckCounter + 1) % 1024;
+    if (ADB_UNLIKELY(_killCheckCounter == 0 && _infos.getQuery().killed())) {
```

Comment:
Is there a reason for not using the standards attributes `[[likely]]/[[unlikely]]`
```suggestion
    if (_killCheckCounter == 0 && _infos.getQuery().killed()) [[ unlikely ]] {
```

Reply:
I have no objections against switching to `[[likely]]` / `[[unlikely]]` in general, but so far we have been using our own macros `ADB_LIKELY` / `ADB_UNLIKELY`, which translate to `__builtin_expect(...)`.
I did not want to introduce potentially other compiler behavior by switching to the now-standardized C++ attributes.
This should be done in a separate PR if required.